docs/scenarios_authoring/Spel.md

```diff
@@ -264,18 +264,89 @@ can be accessed in e.g. in following ways:
 * `#exampleObjects['someNestedObject']['someFieldInNestedObject']`
 * `#exampleObjects['someArrayWithObjects'][0]['someFieldInObjectInArray']`
 
-Every unknown accessed field/element will produce `Unknown` data type which can be further navigated or [cast](#Casting) to a desired type.
+Every unknown accessed field/element will produce `Unknown` data type which can be further navigated or [converted](#type-conversions) to a desired type.
 
 ### Type conversions
 
-It is possible to convert from a type to another type and this can be done by implicit and explicit conversion.
+It is possible to convert or cast from a type to another type and this can be done by implicit and explicit conversion.
+
+#### Explicit conversions
+
+Explicit conversions/casts are available in built-in functions and in utility classes.
+List of built-in functions:
+- `is(className)`/`to(className)`/`toOrNull(className)`
+- `isBoolean`/`toBoolean`/`toBooleanOrNull`
+- `isLong`/`toLong`/`toLongOrNull`
+- `isDouble`/`toDouble`/`toDoubleOrNull`
+- `isBigDecimal`/`toBigDecimal`/`toBigDecimalOrNull`
+- `isList`/`toList`/`toListOrNull`
+- `isMap`/`toMap`/`toMapOrNull`
+
+Functions with the prefix `is` check whether a type can be converted or cast to the appropriate type. Functions with
+the `to` prefix convert or cast a value to the desired type, and if the conversion fails, an exception is propagated
+further. Functions with the `toOrNull` prefix convert or cast a value to the desired type, and if conversion fails, a
+null value is returned.
+The `is`, `to` and `toOrNull` functions are slightly different in that they take a class name as a parameter. They first
+attempt to cast a value to the specified class. If the cast fails and there is a defined conversion to that class, the
+conversion is applied. Below is the list of available conversion to classes:
+- BigDecimal 
+- BigInteger 
+- Boolean 
+- Byte 
+- Charset 
+- ChronoLocalDate 
+- ChronoLocalDateTime
+- Currency 
+- Double 
+- Float 
+- Integer 
+- List 
+- Locale 
+- LocalDate 
+- LocalDateTime 
+- LocalTime 
+- Long 
+- Map 
+- UUID 
+- Short 
+- String 
+- ZoneId 
+- ZoneOffset
+
+Conversions only make sens between specific types. Below is a matrix which shows which types can be converted with each
```

Comment:
sens -> sense.
I think that is worth mention that we are not showing conversions that have no sense in suggestions mechanism

Reply:
Done